### PR TITLE
Add MNavigationDrawer

### PR DESCRIPTION
**Pull request by flyofsky**
### Description of Change ###
This PR is for supporting NavigationDrawer on Tizen.

### API Changes ###
 None

### Platforms Affected ### 
- Tizen

### Behavioral/Visual Changes ###
None

### Before/After Screenshots ### 
selection_007

### PR Checklist ###

- [ ] Has automated tests 
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

File: src/Tizen.NET.MaterialComponents/Components/MNavigationDrawer.cs [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+            _mainContainer.Show();
+            PackEnd(_mainContainer);
+
+            _scrim = new Box(parent)
+            {
+                AlignmentX = -1,
+                AlignmentY = -1,
+                WeightX = 1,
+                WeightY = 1,
+                BackgroundColor = new Color(0, 0, 0, 82)
```
Comment (rookiejava) [inline src/Tizen.NET.MaterialComponents/Components/MNavigationDrawer.cs]: do not use color value directly here.

File: test/MaterialGallery/Gallery/NavigationDrawerPage.cs [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            Box box = new Box(window)
+            {
+                BackgroundColor = Color.White,
+            };
+
+            conformant.SetContent(box);
+            box.Show();
+
+            MNavigationView nv = new MNavigationView(window)
+            {
```
Comment (rookiejava) [inline test/MaterialGallery/Gallery/NavigationDrawerPage.cs]: MNavigationView nv = new MNavigationView(window);

File: src/Tizen.NET.MaterialComponents/Components/MNavigationDrawer.cs (hunk 1 above)
Comment (flyofsky) [reply src/Tizen.NET.MaterialComponents/Components/MNavigationDrawer.cs]: Done

File: test/MaterialGallery/Gallery/NavigationDrawerPage.cs (hunk 2 above)
Comment (flyofsky) [reply test/MaterialGallery/Gallery/NavigationDrawerPage.cs]: Done

**Review by rookiejava — APPROVED** [review]

**Pull request merged**